ID     Step     MaxR |    avgR     stdR | avgS stdS |    objC     etc.
Act FileNotFound when load_model: output/2022-04-25_22-41-34
Critic FileNotFound when load_model: output/2022-04-25_22-41-34
pygame 2.0.1 (SDL 2.0.14, Python 3.6.2)
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "/home/dq/PycharmProjects/ReinforcementLearning/Projects/RMUA/RMUA 2021_V5.100/eRL_demo_SingleFilePPO.py", line 955, in <module>
    demo_discrete_action()
  File "/home/dq/PycharmProjects/ReinforcementLearning/Projects/RMUA/RMUA 2021_V5.100/eRL_demo_SingleFilePPO.py", line 951, in demo_discrete_action
    train_and_evaluate(args)
  File "/home/dq/PycharmProjects/ReinforcementLearning/Projects/RMUA/RMUA 2021_V5.100/eRL_demo_SingleFilePPO.py", line 749, in train_and_evaluate
    if_train_actor)
  File "/home/dq/PycharmProjects/ReinforcementLearning/Projects/RMUA/RMUA 2021_V5.100/eRL_demo_SingleFilePPO.py", line 223, in update_net
    buf_state, buf_action, buf_r_sum, buf_logprob, buf_advantage = self.prepare_buffer(buffer)
  File "/home/dq/PycharmProjects/ReinforcementLearning/Projects/RMUA/RMUA 2021_V5.100/eRL_demo_SingleFilePPO.py", line 269, in prepare_buffer
    r_sum, advantage = self.get_reward_sum(self, buf_len, reward, mask, value, pre_r_sum)
  File "/home/dq/PycharmProjects/ReinforcementLearning/Projects/RMUA/RMUA 2021_V5.100/eRL_demo_SingleFilePPO.py", line 290, in get_reward_sum_gae
    buf_r_sum[i] = buf_reward[i] + buf_mask[i] * pre_r_sum
RuntimeError: expand(torch.cuda.FloatTensor{[2, 1]}, size=[]): the number of sizes provided (0) must be greater or equal to the number of dimensions in the tensor (2)